examples/components/pubsub/kafka.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: dapr.io/v1alpha1
+kind: Component
+metadata:
+  name: kafka-pubsub
```

Comment:
Why not use Redis for this too. The README file will not work without additional instructions to setup Kafka.

Reply:
done.